src/__tests__/orchestrator.integration.spec.ts

```diff
@@ -165,6 +170,35 @@ describe('elastic orchestrator', () => {
       expect(newIndexDocuments).toEqual(oldIndexDocuments)
     })
 
+    it('will trigger a reindex from the index with the same name for a new index', async () => {
+      const existingConfig = ['index-00001.json', 'another-index-00000.json']
+      await manageIndices(client, existingConfig, indexConfigFileFolder)
+
+      const indices = await getExistingIndices(client)
+      const latestIndex = await getMostRecentIndex(
+        client,
+        indices.filter(index => index.startsWith('another-index')),
+      )
+
+      await insertBulk(
+        latestIndex,
+        createBulkPayload(4, value => ({
+          description: `description-${value}`,
+        })),
+      )
+
+      await manageIndices(
+        client,
+        [...existingConfig, 'another-index-00001.json'],
+        indexConfigFileFolder,
+      )
+
+      const oldIndexDocuments = await getIndexDocuments('another-index-00000')
+      const newIndexDocuments = await getIndexDocuments('another-index-00001')
+
+      expect(newIndexDocuments).toEqual(oldIndexDocuments)
```

Comment:
Just to cover all basis we should ensure/assert that `index` and `another-index` are actually different, WDYT?

Reply:
Sure.